README.md

```diff
@@ -23,11 +25,11 @@
 ## Configure a site to be built
 
 1. Navigate to **Settings** for the repo
-2. In **Collaborators**, add `tableflip-deploy` as a collaborator with **write** access
+2. In **Collaborators**, add `tableflip-tabs` as a collaborator with **write** access
 3. in **Webhooks & services** add a new webhook with the following info:
-    * Payload URL: `http://builder.tableflip.io:7777/webhook`
+    * Payload URL: `http://tabs.tableflip.io:7777/webhook`
     * Content type: `application/json`
-    * Secret: **Retrieve from secrets.yaml in infrastructure project**
+    * Secret: **Retrieve from secrets.yaml from tabs-infrastructure project**
```

Comment:
It's not actually _in_ the tabs-infrastructure project as it's in the 1password vault, perhaps "for" instead?